pkg/standalone/list.go

```diff
@@ -166,24 +118,17 @@ func List() ([]ListOutput, error) {
 			listRow := ListOutput{
 				Created: createTime.Format("2006-01-02 15:04.05"),
 				Age:     age.GetAge(createTime),
-				PID:     proc.Pid(),
+				PID:     cliPID,
 			}
 
-			// Now we use sidecar into to decorate with more info (empty, if sidecar is down).
-			run, ok := cliToSidecarMap[proc.Pid()]
-			if ok {
-				listRow.AppID = run.appID
-				listRow.HTTPPort = run.httpPort
-				listRow.GRPCPort = run.grpcPort
-				listRow.AppPort = run.appPort
-				listRow.MetricsEnabled = run.enableMetrics
-				listRow.Command = utils.TruncateString(run.appCmd, 20)
-			}
+			listRow.AppID = appID
+			listRow.HTTPPort = httpPort
+			listRow.GRPCPort = grpcPort
+			listRow.AppPort = appPort
+			listRow.MetricsEnabled = enableMetrics
+			listRow.Command = utils.TruncateString(appCmd, 20)
 
-			// filter only dashboard instance
-			if listRow.AppID != "" {
```

Comment:
@sk593 How is it that removing this check but still the filter dashboard instance from list is still working? What has changed there? @pkedy @wcs1only 
https://github.com/dapr/cli/pull/763/files#diff-40b8b5acdd5260df8de205a924b446eb2483901d4e4c30c6a23932d3223e99e9R117

Reply:
The dashboard instance runs as a `dapr` process. The changes look for `daprd` sidecar processes and then attach the parent `dapr` application (if it exists). Since the dashboard doesn't have a `daprd` sidecar, it's not picked up